sql/moz-fx-data-shared-prod/org_mozilla_firefox_derived/events_daily_v1/metadata.yaml

```diff
@@ -1,21 +0,0 @@
-# Generated by bigquery_etl/events_daily/generate_queries.py
-
----
-friendly_name: 'org_mozilla_firefox Events Daily'
-description: >
-  Packed event representation with one-row per-client
-owners:
-  - frank@mozilla.com
-labels:
-  application: org_mozilla_firefox
-  schedule: daily
-  incremental: true
-scheduling:
-  dag_name: bqetl_fenix_event_rollup
```

Comment:
All the queries that referenced `bqetl_fenix_event_rollup` got removed and will be generated after the DAG generation step. This will result in the DAG having to tasks.